strawberry/cli/commands/import_schema.py

```diff
@@ -0,0 +1,116 @@
+import re
+import os
+from os.path import abspath
+
+import click
+
+from graphql.language import parse
+from graphql.error import GraphQLError
+
+from ...utils.sdl_transpiler import render_template
+from ...utils.str_converters import to_snake_case
+
+
+# Regex pattern to extract import statement
+GQL_IMPORT_REGEX = r"#import \"(.*\.gql|.*\.graphql)\""
+GQL_SCHEMA_TYPE = r"(?si)^Schema.*?}"
+
+
+def set_chdir(fn):
+    """ Helps resolve relative imports based on paths passed as arguments """
+
+    def wrapper(*args):
+        path, imported = args
+        previous_chdir = os.getcwd()
+        os.chdir(os.path.dirname(path))
+        paths = fn(path, imported)
+        os.chdir(previous_chdir)
+        return paths
+
+    return wrapper
+
+
+@click.command("import-schema", short_help="Parses SDL file to python dataclasses")
+@click.argument("schema", type=str, required=True)
+@click.option("-O", "--output", default="", type=str)
+@click.option("-R", "--recursive", default=True, type=bool)
+@click.option("-M", "--multifile", default=False, type=bool)
+def import_schema(schema, output, recursive, multifile):
```

Comment:
> let's simplify this, let's always output the generated file to shell, so instead of creating files directly one will have to use output redirection, like this:
> 
> ```shell
> strawberry import-schema example.graphql > output.py
> ```
> 
> I think it is better to allow the caller to decide what to do with the output :)
---
Yeah in retrospect it might be overkill, it was an initial idea which I aborted pretty quickly but haven't removed it, so I agree with simplification, don't speak windows much but any idea if a redirection can be done in MS environment. I'll do it sometime this week.